.travis.yml

```diff
@@ -19,7 +19,7 @@ before_install:
 
 install:
   - docker build --build-arg BUILD_COMMIT=$TRAVIS_COMMIT --target production -t humanconnection/nitro-web .
-  - docker-compose -f docker-compose.yml -f docker-compose.travis.yml up -d
+  - docker-compose -f docker-compose.yml up -f docker-compose.travis.yml up -d
```

Comment:
no no, the `up` is there already :wink: @mattwr18

Reply:
ahh, right... I thought I had missed it yesterday from your review above... it's still failing the build though